runtime/gear/src/lib.rs

```diff
@@ -89,12 +88,19 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     impl_name: create_runtime_str!("gear-node"),
     apis: RUNTIME_API_VERSIONS,
     authoring_version: 1,
```

Comment:
We also can change ids in specs now to proper ones #1303 ?

Reply:
done